src/graphql/mutations/InteractiveMessage.ts

```diff
@@ -4,9 +4,11 @@ export const CREATE_INTERACTIVE = gql`
   mutation createInteractiveTemplate($input: interactiveTemplateInput!) {
     createInteractiveTemplate(input: $input) {
       interactiveTemplate {
-        type
+        id
         label
         interactiveContent
+        type
+        translations
```

Comment:
It looks like we are duplicating whole translations in the DB. Am I missing something?

Reply:
Not sure what you mean by duplicating the translations here 😅.